File: Leetcode/34-find-first-and-last-position-of-element-in-sorted-array/find-first-and-last-position-of-element-in-sorted-array.py
class Solution(object):
    def searchRange(self, nums, target):
        lb = bisect.bisect_left(nums, target)
        ub = bisect.bisect_right(nums, target)
        if lb == ub:
            return [-1, -1]
        return [lb, ub - 1]
        

File: Leetcode/34-find-first-and-last-position-of-element-in-sorted-array/find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& n, int target) {
        int ub=upper_bound(n.begin(),n.end(),target)-n.begin();
        int lb=lower_bound(n.begin(),n.end(),target)-n.begin();
        if(ub==lb) return {-1,-1};
        return {lb,ub-1};
    }
};